pkg/network/openshift_sdn.go

```diff
@@ -123,8 +123,15 @@ func fillOpenShiftSDNDefaults(conf *netv1.NetworkConfigSpec) {
 		var port uint32 = 4789
 		sc.VXLANPort = &port
 	}
+
+	// MTU is special. If it's not supplied, we infer it from  the node on which
```

Comment:
Maybe rather than being "special", there should be separate methods for "defaults" and "inferred values"?

Reply:
I think you're right. We should even just carry filled defaults from `previous` in all cases.

Reply:
After a bit more thought, I think it makes sense to leave it more-or-less as-is. In a situation where the default changes, I could imagine keeping previous vs. using the new default needs to be done on a case-by-case basis.